packages/debugger-extension/src/index.ts

```diff
@@ -271,60 +275,149 @@ const sources: JupyterFrontEndPlugin<IDebugger.ISources> = {
 const variables: JupyterFrontEndPlugin<void> = {
   id: '@jupyterlab/debugger-extension:variables',
   autoStart: true,
-  requires: [IDebugger, ITranslator],
+  requires: [
+    IDebugger,
+    IDebuggerHandler,
+    ITranslator,
+    IRenderMimeRegistry,
```

Comment:
Probably in most cases and applications the rendermime registry will be available, but wondering whether having it `optional` would also work? So rich rendering is enabled only if `IRenderMimeRegistry` is available?

Reply:
Yes we can make it optional.